### PR TITLE
simplify tccp update

### PR DESCRIPTION
**Pull request by xh3b4sd**
I plan to get the master management in the tccp resource a bit more straight. We also have a EBS service implementation which could be split a bit. I am not 100% sure yet on the primitives because there is some EBS Volume handling in another resource as well but this can probably be a bit better. Step by step.

### CODE REVIEW

File: service/controller/v25/resource/tccp/update.go [hunk 1]
```diff
@@ -213,16 +205,17 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
...
+			templateBody, err := r.newTemplateBody(ctx, cr, desiredStackState)
 			if err != nil {
 				return StackState{}, microerror.Mask(err)
 			}
 
 			updateState := StackState{
-				Name:             desiredStackState.Name,
-				ShouldScale:      true,
-				ShouldUpdate:     false,
-				UpdateStackInput: updateStackInput,
```
Comment (kopiczko) [inline service/controller/v25/resource/tccp/update.go]: Can `UpdateStackInput` also be removed from `StackState` struct definition?

Comment (xh3b4sd) [reply service/controller/v25/resource/tccp/update.go]: For sure. There will happen more. The whole struct might disappear when I find a way to use the simple resource interface. One step at a time.

Comment (xh3b4sd) [reply service/controller/v25/resource/tccp/update.go]: I dropped the property.

**Comment by xh3b4sd** [comment]
Oh yeah, I tested that change on `ginger`.

**Pull request merged**